app/Http/Kernel.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace App\Http;
+
```

Comment:
Did you create this file or did laravel generate it ?

Reply:
laravel generated